### DISCUSSION TITLE
More screenshot options?

### DISCUSSION THREAD

**Discussion by SkullGunner** [andrei-drexler/ironwail#360] [Ideas]
Could the way screenshots are taken be changed so they aren't so large?

By default this Ironwail PNG screenshot is 4.3MB:
![e3m4_pinchy2_2024-11-19_16-45-28](https://github.com/user-attachments/assets/ff2b719f-2fb0-47f2-99f1-862f5ba35e7d)

Running it through tinypng.com shrinks it to 0.6MB and barely changes the quality:
![tinypng](https://github.com/user-attachments/assets/b93d245a-51ad-4e02-bfdb-0c8b205656a0)

Could we copy that type of compression or just allow other image format options like JPG or WEBP?

**Comment by andrei-drexler** [comment]
The screenshot command can take an optional format argument, either `png`, `tga`, or `jpg`. `screenshot jpg` should produce a smaller file.

**Reply by SkullGunner** [reply]
Oh forgive my ignorance, thank you!

**Reply by andrei-drexler** [reply]
No worries, it's a perfectly resonable question/request :)
